sidebars.js

```diff
@@ -85,7 +85,7 @@ const sidebars = {
             {
               type: 'link',
               label: 'Documentation',
-              href: 'https://gobeyondidentity.github.io/bi-sdk-swift/',
+              href: 'https://gobeyondidentity.github.io/bi-sdk-swift/documentation/beyondidentityembedded/',
```

Comment:
I think you can remove all the auto-generated doc links here if you added all of these SDK set up. So ideally the SDK folder would only have these files and we remove the nested individual SDK folders:
```
SDK Setup
Bind Passkey To User
Authenticate with Passkey
Integrate with Auth0 
Integrate with Okta
```
(later we can join the okta and auth0 guides into one file too will different code snippets)

Reply:
Done 👍